examples/tests/valid/intrinsics.test.w

```diff
@@ -0,0 +1,23 @@
+bring fs;
+bring expect;
+bring "./subdir/bar.w" as bar;
+
+let path = "SHOULD_IGNORE";
+let filename = "intrinsics.test.w";
```

Comment:
of course we need another intrinsic for `@filename()`.

Reply:
Why?